inc/saiswitch.h

```diff
@@ -500,6 +500,12 @@ typedef enum _sai_switch_attr_t
        Default no map */
     SAI_SWITCH_ATTR_QOS_TC_AND_COLOR_TO_DSCP_MAP,
 
+    /** The number of default objects created on the switch [sai_uint32_t] */
```

Comment:
@kcudnik @lguohan what is the motivation for the default objects created by the switch? can you provide a use case ? 

Typically there are a lot of attributes/resources available on the ASIC, they are not really created or deleted. Are you referring to OBJECT IDs only or hardware resources.